modules/ai/assets/js/editor/page-content.js

```diff
@@ -77,6 +80,29 @@ const PageContent = (
 		},
 	};
 
+	const codePromptDialogStyleProps = {
+		sx: {
+			...promptDialogStyleProps.sx,
+			...sxStyle,
+		},
+		PaperProps: {
+			onMouseEnter: () => {
+				clearTimeout( timeoutRef.current );
+				setSxStyle( { pointerEvents: 'all' } );
+			},
+			onMouseLeave: () => {
+				clearTimeout( timeoutRef.current );
+				timeoutRef.current = setTimeout( () => {
+					setSxStyle( { pointerEvents: 'none' } );
+				}, 200 );
+			},
+			sx: {
+				...promptDialogStyleProps.PaperProps.sx,
```

Comment:
Is it always exist?
```suggestion
				...promptDialogStyleProps?.PaperProps?.sx,
```

Reply:
yes, it's written as a static param [here](https://github.com/netanelavr/elementor/blob/main/modules/ai/assets/js/editor/page-content.js#L41-L60).